### DISCUSSION TITLE
INPUTNUMBER restricting the user from entering more than 1 decimal values

### DISCUSSION THREAD

**Discussion by aadibryan** [ant-design/ant-design#42747] [Q&A]
<InputNumber
                id="input-number-1"
                stringMode={true}
                status={maxTempError ? 'error' : ''}
                disabled={!maxTempEnabled}
                defaultValue={maxTemp}
                value={maxTemp}
                size="small"
                bordered
                type="number"
                formatter={userInputFormatter}
                // precision={1}
                onChange={(value: any) => onInputChange(value, 'maxTemp')}
              />

How to restrict the user from entering more than 1 decimal values . the value of the input should always not exceed the one digit decimal place.

**Comment by bombillazo** [comment]
Hello, I am also interested in limiting the amount of decimal places in an input to 2 , how is that achieved with the `InputNumber` component?

**Reply by aadibryan** [reply]
I ended up using eventListener to achieve this
